UltimateTeam.Toolkit/Models/AuctionResponse.cs

```diff
@@ -2,13 +2,13 @@
 
 namespace UltimateTeam.Toolkit.Models
 {
-    public class AuctionResponse
+    public class AuctionResponse : IListResponse
```

Comment:
Why are 2 properties `virtual` here? Why should it be possible to override them?